Domains/QuantityTakeoffsAspects/QuantityTakeoffsAspects.ecschema.xml

```diff
@@ -73,11 +78,9 @@
         <ECEnumerator value="10" displayLabel="Narrow View w/ Louver"/>
     </ECEnumeration>
 
+    <!-- TODO: Does BIS have a better home for this Energy-performance data? -->
```

Comment:
rather than TODO in schema, should be an item in some backlog?

Reply:
Created an issue instead.